README.md

```diff
@@ -128,15 +143,9 @@ Latest Packages
 | qtmultimedia | Qt Multimedia | 2021-08-30 |
 | qtscript | Qt Script Tools | 2021-08-30 |
 
-Chat with us!
--------------
-<p><em>Discord is not currently syncing messages with Slack</em></p>
-<p><a href="https://join.slack.com/t/chromebrew/shared_invite/enQtNDA2MTQ5ODQ3NDc2LTA0ZmJlMGFmNmZhOTYwNDE3ZDY0NDA0MWI0OTE3MzJkYTQxN2UxMWQ1YTEzOWFmNTliMGM4NDZjYzY2NjczZGE" target="_blank"><img src="/images/slack.png" alt="Slack Invite" />
-<a href="https://discord.gg/QRrzBXN" target="_blank"><img src="/images/discord.png" alt="Discord Invite" /></a></p>
-
 License
 -------
 
-Copyright 2013-2021 Michal Siwek and [all the awesome contributors](https://github.com/skycocker/chromebrew/graphs/contributors).
+Copyright 2013-2022 Michal Siwek and [all the awesome contributors](https://github.com/skycocker/chromebrew/graphs/contributors).
```

Comment:
@skiman6010: This is your responsibility.  Where were you? :)

Reply:
Hey! Someone stole my job!